keystore-manager.sh

```
@@ -412,45 +1251,71 @@ update_fastlane_config() {
-        # This keeps the file structure intact while only changing the values
-        sed -i.bak \
-            -e "s|default_store_file:.*|default_store_file: \"$keystore_name\",|" \
-            -e "s|default_store_password:.*|default_store_password: \"$keystore_password\",|" \
-            -e "s|default_key_alias:.*|default_key_alias: \"$key_alias\",|" \
-            -e "s|default_key_password:.*|default_key_password: \"$key_password\"|" \
-            "$config_file"
-
-        # Remove the backup file
-        rm -f "$config_file.bak"
+        # Create a temporary file for the updated content
+        local temp_file=$(mktemp)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)